pkg/config/redisstore.go

```diff
@@ -18,12 +18,36 @@ import (
 	"fmt"
 	"net/url"
 	"strconv"
+	"strings"
+	"sync"
+	"time"
 
 	"github.com/golang/glog"
+	"github.com/mediocregopher/radix.v2/pubsub"
 	"github.com/mediocregopher/radix.v2/redis"
 )
 
+const (
```

Comment:
Yeah, it wound up being larger than I expected. I think the resulting code is cleaner though, thanks!